### PR TITLE
Fix display of Assist command executions with empty output

### PR DESCRIPTION
**Pull request by justinas**
Partially addresses https://github.com/gravitational/teleport.e/issues/1454 .

After live execution

Screenshot 2023-05-26 at 16 43 07

Coming back to the conversation (when history is fetched)

Screenshot 2023-05-26 at 16 34 26

Now also improved to display when the command exited with a status code (previously said "no output" with no indication that the command failed). This only applies when returning to a previous conversation.
Screenshot 2023-05-29 at 15 26 25

### CODE REVIEW

**Review by ryanclark — APPROVED** [review]
Nice one

**Review by tigrato — APPROVED** [review]

File: web/packages/teleport/src/Assist/Chat/ChatItem/Action/RunAction.tsx [hunk 1]
```diff
@@ -245,11 +270,14 @@ function NodeOutput(props: NodeOutputProps) {
         </LoadingContainer>
       )}
 
-      {props.state.stdout && (
-        <NodeContent>
-          <pre>{props.state.stdout}</pre>
-        </NodeContent>
-      )}
+      {props.state.stdout !== undefined &&
+        (props.state.stdout === '' ? (
```
Comment (gzdunek) [inline web/packages/teleport/src/Assist/Chat/ChatItem/Action/RunAction.tsx:274]: So if `stdout` is `undefined` we want to show `<NodeContent>`?

IMO it should be `!props.state.stdout`
It checks for undefined or empty value.

**Review by gzdunek — APPROVED** [review]
With one comment.

File: web/packages/teleport/src/Assist/Chat/ChatItem/Action/RunAction.tsx (hunk 1 above)
Comment (justinas) [reply web/packages/teleport/src/Assist/Chat/ChatItem/Action/RunAction.tsx:274]: Hmm, this works well in practice. Unless I'm misreading the parentheses/ternaries, this is equivalent to:

```js
if (props.state.stdout !== undefined) {
  if (props.state.stdout === '')
    return "Empty output";
  }
  return <NodeOutput>...</NodeOutput>
} else {
  // props.state.stdout === undefined, display nothing
}
```

Comment (gzdunek) [reply web/packages/teleport/src/Assist/Chat/ChatItem/Action/RunAction.tsx:274]: I didn't notice the parentheses 🤦‍♂️ 
Sorry for the noise.

**Review by gzdunek — APPROVED** [review]

**Pull request merged**